### PR TITLE
serve react build file from the server 

### PR DESCRIPTION
**Pull request by tawfiknasser**
in this PR
we let the route "/" serve the static files , the files inside the build folder

### CODE REVIEW

File: server/routes/index.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
 const express = require('express');
+const path = require('path');
 
 const router = express.Router();
 const voters = require('../model/voters');
 /* GET home page. */
 router.get('/', (req, res) => {
-  res.send('helloww');
+  res.sendFile(path.join(__dirname, '..', '..', 'client/build/index.html'));
```
Comment (MynahMarie) [inline server/routes/index.js]: This is not correct. This path shouldn't be there. The only routes you need to have in your server are the ones that your client is going to query for data. React will render the home page and you will need to query the server for authentication, which should be set to a different route.

File: server/app.js [hunk 2]
```diff
@@ -11,7 +11,9 @@ app.use(logger('dev'));
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 app.use(cookieParser());
-app.use(express.static(path.join(__dirname, 'public')));
+
+app.use(express.static(path.join(__dirname, '..', 'client/build')));
```
Comment (MynahMarie) [inline server/app.js]: This should be the case only when you are in production environment. You will not build your project every time that you are running it on a dev environment. Here is an example of how it should be done:

```
if (process.env.NODE_ENV === 'production') {
    app.use(express.static('client/build'));

    const path = require('path');
    app.get('*', (req, res) => {
        res.sendFile(path.resolve(__dirname, 'client', 'build', 'index.html'));
    });
}
```

File: server/routes/index.js (hunk 1 above)
Comment (tawfiknasser) [reply server/routes/index.js]: we are not using react router, i understand that this approach is usually for setting up react router but we are not doing this.

**Comment by tawfiknasser** [comment]
We refactored the code and used react router.
first time using the react router hope we used it in right way , despite that it's working as final product.

**Pull request merged**